www/js/desktop_scripts.js

```diff
@@ -34,11 +35,14 @@ if (platform === 'win32') {
   }
 }
 
+const dbSchemaPath = schemaPath =>
+  !database[dbPlatform].dbSchema || path.resolve(schemaPath, database[dbPlatform].dbSchema);
+
 const dbPath = path.resolve(userDataPath, database[dbPlatform].dbName);
-const dbSchema = path.resolve(userDataPath, database[dbPlatform].dbSchema);
+const dbSchema = dbSchemaPath(userDataPath);
```

Comment:
that really interesting, this would evaluate to `true` for sqlite. you then pass that to `fs.existsSync()` which only takes `path <string> | <Buffer> | <URL>` [DOCS](https://nodejs.org/docs/latest-v8.x/api/fs.html#fs_fs_existssync_path). I'm wondering if that would have any issues down the road. its returning false now, so Its fine for now.